test/commands/config/set.test.js

```diff
@@ -0,0 +1,273 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { expect, test } from '@oclif/test';
+import * as config from '../../../src/utils/config';
+import * as print from '../../../src/utils/print';
+
+describe('config:set', () => {
+	const defaultConfig = {
+		api: {
+			network: 'main',
+			nodes: ['http://localhost:4000'],
+		},
+		json: false,
+	};
+
+	const printMethodStub = sandbox.stub();
+	const defaultDir = './someDir';
+	const setupStub = () =>
+		test
```

Comment:
Is there a way to use this for the base command?

Reply:
I far as I understand `base command` is not a command, so i wasn't sure how to use this

Reply:
What if you just don't use the `.command` method, and go straight to a `.it`? After all doesn't this just extend fancytest?

Reply:
mmm yea ill try it out